package.json

```diff
@@ -58,28 +58,28 @@
     "babel-jest": "28.x.x",
     "babel-loader": "8.x.x",
     "babel-preset-react-app": "^10.0.1",
-    "copy-webpack-plugin": "^6.0.3",
+    "copy-webpack-plugin": "11.x.x",
     "dotenv": "^16.0.1",
-    "error-overlay-webpack-plugin": "1.0.0",
+    "error-overlay-webpack-plugin": "1.1.0",
```

Comment:
Not sure if this is helpful but this update used to give us some errors (more info [here](https://github.com/gregberge/error-overlay-webpack-plugin/pull/95))and `v1.0.0` seems to work with webpack v4. With this migration to v5, can confirm that this isn't an issue anymore 🙌 .

Reply:
Thank you !